Assets/Placenote/LibPlacenote/LibPlacenote.cs

```diff
@@ -492,11 +492,27 @@ private void Init ()
 		initParams.appBasePath = Application.streamingAssetsPath + "/Placenote";
 		initParams.mapPath = Application.persistentDataPath;
 
-        	#if !UNITY_EDITOR
+        #if !UNITY_EDITOR
 		PNInitialize (ref initParams, OnInitialized, IntPtr.Zero);
 		#endif
 	}
 
+
+	/// <summary>
+	/// Initializes the LibPlacenote SDK singleton class.
+	/// </summary>
+	public void Shutdown ()
+	{
+		#if UNITY_EDITOR
+		mInitialized = false;
```

Comment:
You might need to call StopSession() as well when Shutting down at least for UNITY_EDITOR. StopSession stops all coroutines which are used to simulate the OnPose function.